x/simulation/account.go

```diff
@@ -38,13 +37,8 @@ func RandomAccounts(r *rand.Rand, n int) []Account {
 		// don't need that much entropy for simulation
 		privkeySeed := make([]byte, 15)
 		r.Read(privkeySeed)
-		useSecp := r.Int63()%2 == 0
```

Comment:
Is this related to the cahce change?

Reply:
Not directly, but was needed for initial benchmarking which I henceforth removed. So technically I can undo this change, but we only support secp256k1 keys anyway. Might as well keep it.